lib/index.js

```diff
@@ -53,12 +53,13 @@ class Rooftop {
       }
 
       if (transformFn === true) transformFn = transform
-      if (transformFn === false) transformFn = (x) => x
+      else if (transformFn === false) transformFn = (x) => x
```

Comment:
I agree it is much more efficient tho the logic is flawed when we pass a custom function (for a single liner)
```javascript
transformFn = Boolean(transformFn) ? transform : (x) => x
// transformFn = true ... returns transform
// transformFn = Function is true ... returns transform
// transformFn = false ... returns (x) => x
```

We could do it like this:
```javascript
if (transformFn) transformFn = transform
else transform = Boolean(transformFn) ? transformFn : (x) => x
// transformFn = true ... returns transform
// transformFn = Function is true ... returns transformFn
// transformFn = false ... returns (x) => x
```